dspace/config/spring/api/core-services.xml

```diff
@@ -45,7 +45,17 @@
     <bean class="org.dspace.content.BundleServiceImpl"/>
     <bean class="org.dspace.content.CommunityServiceImpl"/>
     <bean class="org.dspace.content.InstallItemServiceImpl"/>
-    <bean class="org.dspace.content.ItemServiceImpl"/>
+    <bean class="org.dspace.content.ItemServiceImpl">
+        <property name="nonCopyableMetadataFields">
+            <set>
+                <value>dc.date.accessioned</value>
+                <value>dspace.entity.type</value>
+                <value>dc.identifier.uri</value>
+                <value>dc.provenance</value>
```

Comment:
This field doesn't exist in DSpace out-of-the-box. So it shouldn't be listed here.